ui/site/src/socket.js

```diff
@@ -145,15 +147,15 @@ lichess.StrongSocket = function(url, version, settings) {
     return JSON.stringify(data);
   };
 
-  var handle = function(m) {
+  var handle = function(m, idx) {
     if (m.v) {
       if (m.v <= version) {
         debug("already has event " + m.v);
         return;
       }
-      if (m.v > version + 1) {
+      if (m.v > version + 1 && !idx) {
```

Comment:
This looks like it could be an accidental functional change, because this branch is not just for monitoring, also for the `return` a couple lines below. What's the intent?

Reply:
when we get a batch of versioned events they'll all be mismatched, so only operate on the first one of the batch.  I looked at the scala and batches appear to only batch versioned messages as far as I can tell

Reply:
makes sense, but now we wouldn't skip the later messages in a batch, right?

Reply:
Err. Glad I sent this as a PR